config/initializers/airbrake.rb

```diff
@@ -9,7 +9,7 @@
       config.host = begin
         host_uri = URI::parse(ENV['AIRBRAKE_HOST'])
         port = ENV['AIRBRAKE_PORT'].presence&.to_i || host_uri.port
-        uri_builder = port == 443 ? URI::HTTPS : URI::HTTPS
+        uri_builder = URI::HTTPS
```

Comment:
I wonder if this was a typo, and we intended to have some `URI::HTTP` support at some point?

Reply:
I can change it back to set `URI:HTTP` as the else case if we'd like.